### ISSUE TITLE
Vue.js: 'unable to resolve path to module' if using the `@` alias to `src` folder

### ISSUE THREAD

**Issue by ghnoob** [AtomLinter/linter-eslint#1438]
## Issue Type
Bug

## Issue Description
The extension is giving false positives when using Vue's default alias to the `src` folder, but it's working properly with normal relative paths.

![issue screenshot](https://user-images.githubusercontent.com/58740322/127255114-5cd1383c-8691-460b-9537-bdb17eec9585.png)

### My config files
`eslintrc.js`
```js
module.exports = {
  root: true,

  env: {
    node: true,
  },

  extends: [
    'plugin:vue/vue3-essential',
    '@vue/airbnb',
  ],

  parserOptions: {
    parser: '@babel/eslint-parser',
  },

  rules: {
    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
    'no-underscore-dangle': 'off',
    'no-param-reassign': 'off',
  },

  overrides: [
    {
      files: [
        '**/__tests__/*.{j,t}s?(x)',
        '**/tests/unit/**/*.spec.{j,t}s?(x)',
      ],
      env: {
        jest: true,
      },
    },
  ],
};
```
`jsconfig.json`
```json
{
  "compilerOptions": {
    "target": "es5",
    "module": "esnext",
    "baseUrl": "./",
    "moduleResolution": "node",
    "paths": {
      "@/*": [
        "src/*"
      ]
    },
    "lib": [
      "esnext",
      "dom",
      "dom.iterable",
      "scripthost"
    ]
  }
}
```
`vue.config.js`
```js
module.exports = {
  lintOnSave: true,
};
```

## Bug Checklist
- [x] Restart Atom
- [x] Verify the `eslint` CLI gives the proper result, while `linter-eslint` does not
- [x] Paste the output of the `Linter Eslint: Debug` command from the Command Palette below

```
Atom version: 1.57.0
linter-eslint version: 8.6.6
ESLint version: 7.31.0
Hours since last Atom restart: 0
Platform: linux
Using local project ESLint from: /home/rodrigo/Documentos/GitHub/laboratorio-3-final/node_modules/eslint
Current file's scopes: [
  "text.html.vue",
  "source.js.embedded.html",
  "meta.import.js",
  "string.quoted.single.js"
]
linter-eslint configuration: {
  "scopes": [
    "source.js",
    "source.jsx",
    "source.js.jsx",
    "source.flow",
    "source.babel",
    "source.js-semantic",
    "text.html.vue"
  ],
  "lintHtmlFiles": false,
  "autofix": {
    "fixOnSave": false,
    "ignoreFixableRulesWhileTyping": false,
    "rulesToDisableWhileFixing": []
  },
  "global": {
    "useGlobalEslint": false,
    "eslintrcPath": "",
    "globalNodePath": ""
  },
  "disabling": {
    "disableWhenNoEslintConfig": true,
    "rulesToSilenceWhileTyping": []
  },
  "advanced": {
    "disableEslintIgnore": false,
    "disableFSCache": false,
    "showRuleIdInMessage": true,
    "eslintRulesDirs": [],
    "localNodeModules": ""
  }
}
```

**Comment by UziTech** [comment]
This extension just runs eslint and displays the results. If the result is wrong you will need to create an issue on the eslint repo.

**Comment by ghnoob** [comment]
That's weird. It works fine in the CLI and with other editors.

**Comment by UziTech** [comment]
Could it be a different version of eslint is being used?

The built in version is old.

**Comment by ghnoob** [comment]
As far as i know the extensions in both Atom and VSCode should use ESLint version 7.31.0 since it's the one installed in my `node_modules` folder and i don't have any global installation. Am i losing something?

**Comment by ghnoob** [comment]
Update: I tested all the presets from Vue CLI and this issue is only present when using Airbnb config.

**Comment by UziTech** [comment]
If you are using eslint v8 you should use [linter-eslint-node](https://atom.io/packages/linter-eslint-node).
